facebookads/api.py

```diff
@@ -65,6 +69,10 @@ def __init__(self, body=None, http_status=None, headers=None, call=None):
         self._headers = headers or {}
         self._call = call
 
+        # ensure all headers keys are upper-case
+        _headers = dict((key.upper(), value) for (key, value) in headers.items())
```

Comment:
This looks like a breaking change, the headers are getting uppercased while they're exposed by `def headers(self):` below.

We have some code that actually depends on the current behaviour. I think this might need a bit more consideration